src/platform_impl/macos/view.rs

```diff
@@ -463,7 +464,16 @@ declare_class!(
             }
             let was_in_preedit = self.state.ime_state == ImeState::Preedit;
 
-            let characters = get_characters(event, false);
+            // Get the characters from the event.
+            let ev_mods = event_mods(event);
+            let ignore_alt_characters = match self.window().option_as_alt() {
+                OptionAsAlt::OnlyLeft if event.lalt_pressed() => true,
+                OptionAsAlt::OnlyRight if event.ralt_pressed() => true,
+                OptionAsAlt::Both if ev_mods.alt() => true,
+                _ => false,
+            } && !ev_mods.ctrl();
```

Comment:
Still think we may want to check `!ev_mods.logo()` as well - at least that's what I think Terminal.app does?

Reply:
But the `logo` shouldn't modify input chars at all?

Reply:
I guess it would make sense to have `logo` just for consistency, since otherwise you tell to modify logo.